nixos/modules/tasks/filesystems.nix

```diff
@@ -186,7 +186,7 @@ let
       isBindMount = fs: builtins.elem "bind" fs.options;
       skipCheck = fs: fs.noCheck || fs.device == "none" || builtins.elem fs.fsType fsToSkipCheck || isBindMount fs;
       # https://wiki.archlinux.org/index.php/fstab#Filepath_spaces
-      escape = string: builtins.replaceStrings [ " " "\t" ] [ "\\040" "\\011" ] string;
+      escape = string: builtins.replaceStrings [ " " "\t" ] [ "\\x20" "\\x9" ] string;
```

Comment:
we have functions to make this generic, instead of a bespoke replace in `nixos/lib/utils.nix`.

see e.g. `escapeSystemdPath`.

Reply:
Alright, sure, but how do you specifically use `utils.escapeSystemdPath` specifically on systemd units that are generated outside of the NixOS generation, completely away from Nix? You have a systemd component that takes whatever's in the fstab and uses it to generate mount units. You can't make Nix give it something else that's not in the fstab already